arboretum/permissions/README.md

```diff
@@ -94,6 +94,55 @@ how to include the fetchers and checks from this library in your downstream proj
 
 ## Checks
 
+### Organization Integrity (Repository Permissions)
+
+* Class: [GithubOrgPermissionsCheck][org-permissions-check]
+* Purpose: Control the access to Github repositories containing source code.
```

Comment:
This is misleading.  Checks don't control anything.  This check's purpose is to validate a condition is not violated.  State what is being validated.  The check doesn't care whether the repos it is looking at contain source code or not.

Reply:
Done